docker/api/src/Domain/Order/User_order.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/** @noinspection PhpPropertyOnlyWrittenInspection */
+
+namespace App\Domain\Order;
+
+use App\Domain\Product\Product;
+use App\Domain\User\User;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\JoinColumn;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\Table;
+use JetBrains\PhpStorm\ArrayShape;
+use JsonSerializable;
+
+#[Entity, Table(name: 'user_order')]
```

Comment:
```suggestion
#[Entity, Table(name: 'order')]
```

Reply:
It is more simple to just use order for table name. The class name User_order and file name User_order.php should probably be switched to just Order as well